packages/marketing-modal/src/MarketingModal/MarketingModal.tsx

```diff
@@ -79,6 +81,7 @@ const MarketingModal = ({
         >
           {linkText}
         </Link>
+        {disclaimer && <div className={disclaimerStyles}>{disclaimer}</div>}
```

Comment:
This isn't part of your PR, but the above `Link` on L76 should be either wrapped in `Body` or we should set a default `font-size` on the modal